### PR TITLE
[Matter_yamltests] Add to flake8 in workflow and fix python files (part #25193) 

### PR DESCRIPTION
**Pull request by DamMicSzm**
This pull request is a part of #25193 ( part fix python files in Matter_yamltests )

### Problem

Python files need prevent things like syntax errors, typos, bad style, etc... it saves time for reviewing your code. Many python files needed bug fixes.

### Changes

Fix all python files in Matter_yamltests where linter find problem. Adding Matter_yamltests to check with flake8.

### Testing

CI will test and maybe some another manual testing

### CODE REVIEW

File: scripts/tests/chiptest/yamltest_with_chip_repl_tester.py [hunk 1]
```diff
@@ -15,22 +15,23 @@
...
+import builtins
 import logging
 import os
 import tempfile
 import traceback
 
 # isort: off
 
-from chip import ChipDeviceCtrl  # Needed before chip.FabricAdmin
+from chip import ChipDeviceCtrl  # noqa: F401 # Needed before chip.FabricAdmin
```
Comment (bzbarsky-apple) [inline scripts/tests/chiptest/yamltest_with_chip_repl_tester.py:26]: What does that noqa bit do?

**Review by bzbarsky-apple — APPROVED** [review]

File: scripts/tests/chiptest/yamltest_with_chip_repl_tester.py (hunk 1 above)
Comment (DamMicSzm) [reply scripts/tests/chiptest/yamltest_with_chip_repl_tester.py:26]: ChipDeviceCtrl needed before chip.FabricAdmin but is not used directly in this file. Flake8 reports the problem and noqa allows you to bypass this line.

Comment (bzbarsky-apple) [reply scripts/tests/chiptest/yamltest_with_chip_repl_tester.py:26]: I think that info should probably go into the comment (that F401 is "Module imported but unused", etc).

**Comment by woody-apple** [comment]
ping @DamMicSzm

File: scripts/tests/chiptest/yamltest_with_chip_repl_tester.py [hunk 2]
```diff
@@ -40,11 +42,13 @@
 
     from matter_idl import matter_idl_types
 
+    __ALL__ = (matter_idl_types)
```
Comment (arkq) [inline scripts/tests/chiptest/yamltest_with_chip_repl_tester.py:45]: Why this?

**Pull request merged**